docs/user/tutorials/install_epinio_on_rancher_desktop.md

```diff
@@ -0,0 +1,34 @@
+# Rancher Desktop configuration
+
+## Rancher Desktop Prerequisites
+
+* Running on Windows requires Windows Subsystem for Linux (WSL) which is automatically installed by Rancher Desktop
+* Epinio v0.1.1 has been tested with Rancher Desktop incl. kubernetes version v1.21.5
+
+### Install Rancher Desktop
+
+Install the [latest version](https://github.com/rancher-sandbox/rancher-desktop/releases) from Rancher Desktop for your operating system
+
+## Setup Kubernetes
+
+When running Rancher Desktop for the first time wait until the initialization is completed. Make sure that a supported Kubernetes version is selected under `Kubernetes Settings`, e.g. **v1.21.5**
```

Comment:
I was thinking about it, the problem is that the higher version 1.22.2 does not work atm because of #833. Should we maintain a document were we list the supported Kubernetes versions and reference this here to reduce duplication?

Reply:
I think "versions newer than" is ok. "versions up to" is also ok. But cherry-picked versions is not. If something doesn't work we should probably know why unless it's a too-old version and nobody cares anymore.